superduperdb/components/dataset.py

```diff
@@ -12,8 +12,10 @@
 from superduperdb.base.datalayer import Datalayer
 from superduperdb.base.document import Document
 from superduperdb.components.component import Component
+from superduperdb.misc.annotations import PublicAPI
 
 
+@PublicAPI(stability='stable')
```

Comment:
Great, I think lower case is better for decorators: `@public_api`.

Reply:
done